recipes/openfst/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,6 @@
+#include <fst/fstlib.h>
+
+int main() {
+  fst::StdVectorFst v_fst;
```

Comment:
I would print something like `v_fst.size()` just to make sure unused object is not removed. not critical.

Reply:
will change